tests/applications/test_nexus_helpers.py

```diff
@@ -18,22 +20,25 @@
     RandomEV44Generator,
     random_ev44_generator,
 )
+from beamlime.applications.daemons import fake_event_generators
+from tests.applications.data import get_path
 
 
 @pytest.fixture
-def ymir_detectors_container() -> NexusContainer:
-    from tests.applications.data import get_path
-
-    return NexusContainer.from_template_file(get_path('ymir_detectors.json'))
+def ymir():
```

Comment:
I made the name specific because this one is modified from the original `ymir` template.
Can we keep the `detectors` in the name?